eslint.config.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const eslintConfigESLint = require("eslint-config-eslint/cjs");
+const globals = require("globals");
+
+module.exports = [
+    ...eslintConfigESLint,
```

Comment:
```suggestion
    ...eslintConfigESLint,
    {
        ignores: ["coverage/"]
    },
```

Since the lint command is changed to `eslint .`, directories that shouldn't be linted should be ignored.